docs/dev.md

```diff
@@ -93,21 +93,29 @@ Your project must have a Dockerfile to use dev mode in container mode. A sample
 
 Note: Open Liberty images with the `kernel-slim` tag are not supported with dev mode at this time.
 
-Dev mode works with a temporary modified copy of your Dockerfile to allow for hot deployment during development. When dev mode starts up, it pulls the latest version of the parent image defined in the Dockerfile, builds the container image, then runs the container. Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits, the container is stopped and deleted, and the logs are preserved in the directory mentioned above.
+Dev mode works with a temporary, modified copy of your Dockerfile to allow for hot deployment during development as detailed below. When dev mode starts up, it pulls the latest version of the parent image defined in the Dockerfile, builds the container image, then runs the container. Note that the context of the `docker build` command used to generate the container image is the directory containing the Dockerfile. When dev mode exits, the container is stopped and deleted, and the logs are preserved in the directory mentioned above.
+
+Dev mode modifies the Dockerfile in order to allow for hot deployment. Hot deployment is possible because the application is installed as a loose application WAR. This method uses a file type of `.war.xml` which is functionally equivalent to the `.war` file. Dev mode only supports the application under development in the current project so to avoid application conflicts, dev mode removes all Dockerfile commands that copy or add a `.war` file.
```

Comment:
The first sentence in this paragraph is almost a repeat of the first sentence in the previous paragraph. Maybe remove it and start with a modified second sentence: "Hot deployment is made possible because...."

Reply:
Ok!